files/brews/hub.rb

```diff
@@ -0,0 +1,22 @@
+require 'formula'
+
+class Hub < Formula
+  VERSION = "2.2.0-rc1"
+
+  homepage "http://hub.github.com/"
+  url "https://github.com/github/hub/releases/download/v#{VERSION}/hub_mac_amd64_v#{VERSION}.tar.gz"
+  sha1 "413ce80d32f6313cef010e0edb1e664e1fff7427"
+
+  def install
+    bin.install "hub"
+    man1.install Dir["man/*"]
+    bash_completion.install "etc/hub.bash_completion.sh"
+    zsh_completion.install "etc/hub.zsh_completion" => "_hub"
+  end
+
+  test do
+    HOMEBREW_REPOSITORY.cd do
+      assert_equal true, !!(shell_output("#{bin}/hub version").strip =~ /hub version #{VERSION}/)
```

Comment:
No need for strict boolean comparison and regex. This will do:

```rb
assert shell_output("#{bin}/hub version").split("\n").include?("hub version #{VERSION}")
```